grab-workspace-deps.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
```

Comment:
Is there a reason to not just write these operation directly in the docker file?

Also, consider adding comments at the top of this file (if you plan to still have it) about why we have the file, and not so much about what it does (though keep those comments)

Something along the lines of:

"Since different packages / workspace require different dependencies we allow users to specify dependencies required using a scripts. A scripts is used to handle building from source"

Reply:
It's true, this contents of this script can now be directly used in Dockerfile. Will be removing this script.